charts/incubator/dendrite/values.yaml

```diff
@@ -10,7 +10,7 @@ image:
   # -- image repository
   repository: matrixdotorg/dendrite-monolith
   # -- image tag
-  tag: "v0.6.2"
+  tag: "v0.6.3"
```

Comment:
Could it make sense to replace all of these with the following snippet?

```yaml
  # -- image tag
  # @default -- chart.appVersion
  tag:
```

That way, the version gets read from the chart metadata by default, and it saves us having to specify it so many times.

Reply:
Yeah I agree that is much nicer. Because of the way I am using Subcharts, there was a little extra work to pass the app version through to those but nothing too major :)